_schemaV2.graphql

```diff
@@ -5120,6 +5120,9 @@ type Money {
 
   # An amount of money expressed in major units (like dollars).
   major: Float!
+
+  # An amount of money converted to USD major units (dollars).
+  toUSD: Float!
```

Comment:
Wondering if rather than an extra field, `major` and `minor` can get arguments passed to it which might return USD? like `major(convertTo: "USD")` or something.

Reply:
Nice! That sounds like a better and more future-proof api design.

I took your suggestion, but with two caveats:
- Only applied to `major`, since major-to-major conversion is the only supported use case now
- Raises an error if you try to convert to something other than USD, since that's the only currency we need to support presently (and that's what the exchange rate tables are based on). This field design should allow this to be expanded later if need be.